next.config.js

```diff
@@ -0,0 +1,8 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  images: {
+    domains: ["picsum.photos"],
```

Comment:
Do we have used picsum photos anywhere?
If yes, let's download and add it to our server, so project will not have external dependency.